docs/quickstart.rst

```diff
@@ -186,7 +186,14 @@ the model does not predict that outcome.
     Name: false_negative_rate, dtype: float64
 
 Fairlearn also allows us to quickly plot these metrics from the
-:class:`fairlearn.metrics.MetricFrame`
+:class:`fairlearn.metrics.MetricFrame`. 
+
+.. doctest:: quickstart
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.metrics import false_positive_rate, selection_rate, count
+    >>> from functools import partial
+    >>> zero_div_precision_score = partial(precision_score, zero_division=0)
```

Comment:
I don't think these should be included here, since they might go out of sync with the example if the example changes.

The code snippet bellow doesn't run as is anyway. It requires `y_true` and `y_pred` and the rest of the script to run to work. We can instead add an explicit note that this is a snippet from the example.